rb/lib/selenium/webdriver/bidi/browsing_context.rb

```diff
@@ -73,6 +73,35 @@ def get_tree(max_depth: nil)
           )
         end
 
+        def print_page(**options)
+          print_result = @bidi.send_cmd('browsingContext.print',
+                                        context: @id,
+                                        background: false,
+                                        margin: {
+                                          bottom: options['bottom'] || 1.0,
+                                          left: options['left'] || 1.0,
+                                          right: options['right'] || 1.0,
+                                          top: options['top'] || 1.0
+                                        },
+                                        orientation: 'portrait',
+                                        page: {
+                                          height: options.dig(:page, :height) || 27.94,
```

Comment:
Why so precise default values? Is it something from the spec?

Reply:
Yes, the default values are mentioned in the spec.
https://w3c.github.io/webdriver-bidi/#command-browsingContext-print